src/components/views/DownloadModModal.vue

```diff
@@ -133,10 +132,9 @@ import ProfileModList from '../../r2mm/mods/ProfileModList';
                     } catch (e) {
                         this.$store.commit('error/handleError', R2Error.fromThrownValue(e));
                     }
-                }, async (downloadedMods) => {
-                    await this.downloadCompletedCallback(downloadedMods);
-                    this.$store.commit('download/setIsModProgressModalOpen', false);
                 });
+                this.$store.commit('download/setIsModProgressModalOpen', false);
```

Comment:
Why switch the order? Now the modal is closed before install finishes, and the user might start the game modded before all mods are copied into profile folder.

Reply:
oops, wasn't supposed to.